microhapulator/cli/getrefr.py

```diff
@@ -12,8 +12,7 @@
 
 
 from hashlib import sha1
-from pathlib import Path
-from pkg_resources import resource_filename
+from importlib.resources import files
```

Comment:
The pkg_resources package is also removed in Python 3.12, so it was replaced by importlib.resources.